packages/mui/src/components/crud/show/index.tsx

```diff
@@ -34,6 +34,7 @@ import {
 } from "@components";
 import type { ShowProps } from "../types";
 import { RefinePageHeaderClassNames } from "@refinedev/ui-types";
+import { CircularProgress } from "@mui/material";
```

Comment:
should be imported like that because `@mui` has issues with tree-shaking.

```suggestion
import CircularProgress from "@mui/material/CircularProgress";
```

Reply:
Aaa I see, thanks @alicanerdurmaz! I updated the import as suggested.